src/views/FindOrder.vue

```diff
@@ -104,61 +104,80 @@
+                    {{ order.doclist.docs[0].orderId }}
+                    <p> {{ order.doclist.docs[0].customerName }} </p>
                   </ion-label>
                 </ion-item>
               </div>
 
               <div class="tags">
-                <ion-chip outline>
+                <ion-chip @click="copyToClipboard($filters.getOrderIdentificationId(order.doclist.docs[0].orderIdentifications, 'orderIdentificationTypeId'))"  outline v-if="$filters.getOrderIdentificationId(order.doclist.docs[0].orderIdentifications, 'orderIdentificationTypeId')">
                   <ion-icon :icon="pricetag" />
-                  <ion-label>Shopify ID</ion-label>
+                  <ion-label> {{ $filters.getOrderIdentificationId(order.doclist.docs[0].orderIdentifications, 'orderIdentificationTypeId') }} </ion-label>
                 </ion-chip>
-                <ion-chip outline>
+                <ion-chip outline v-if="$filters.getCustomerLoyalty(order.doclist.docs[0].orderNotes, 'cusotmerLoyaltyOptions')">
                   <ion-icon :icon="ribbon" />
-                  <ion-label>Customer Loyalty Status</ion-label>
+                  <ion-label> {{ $filters.getCustomerLoyalty(order.doclist.docs[0].orderNotes, 'cusotmerLoyaltyOptions') }} </ion-label>
                 </ion-chip>
               </div>
 
               <div class="metadata">
-                <ion-note>Ordered on 7 Jan 2021</ion-note>
-                <ion-badge>Approved</ion-badge>
+                <ion-note> {{ $t("Ordered on") }} {{ $filters.formatUtcDate(order.doclist.docs[0].orderDate, 'YYYY-MM-DDTHH:mm:ssZ') }} </ion-note>
+                <ion-badge> {{ order.doclist.docs[0].orderStatusId }} </ion-badge>
               </div>
             </section>
 
-            <section class="section-grid" @click="() => router.push('/order')">
-              <div v-for="item in 7" :key="item">
+            <section class="section-grid" @click="() => router.push(`/order/${order.doclist.docs[0].orderId}`)">
+              <div v-for="(item, index) in order.doclist.docs" :key="index" :item="item">
                 <ion-card>
                   <ion-item>
                     <ion-thumbnail slot="start">
-                      <Image src="https://cdn.shopify.com/s/files/1/0069/7384/9727/products/test-track.jpg?v=1626255137" />
+                      <Image :src="getProduct(item.productId).mainImageUrl" />
                     </ion-thumbnail>
                     <ion-label>
-                      <p>Brand</p>
-                      Virtual Name
-                      <p>{{ $t("Color") }} : color</p>
-                      <p>{{ $t("Size") }}: size</p>
+                      <p> {{ getProduct(item.productId).brandName ? getProduct(item.productId).brandName : '-' }} </p>
+                      {{ item.virtualProductName }}
+                      <p> {{ $t("Color") }} : {{ $filters.getFeature(getProduct(item.productId).featureHierarchy, '1/COLOR/') }} </p>
```

Comment:
Add a todo comment here to make this dynamic, where feature key and value will both be dynamically pulled from source. Different retailers will have different product feature types so we cannot hardcode this.

Reply:
Added a todo and will handle this in another task.